dojo/engagement/views.py

```diff
@@ -180,9 +180,10 @@ def view_engagement(request, eid):
         eng.save()
 
     add_breadcrumb(parent=eng, top_level=False, request=request)
-
+    findings = Finding.objects.filter(test__engagement=eng, duplicate=False)
```

Comment:
This should check for `settings.ENABLE_DEDUPLICATION` so that users can control whether they want to filter these out or not